org.sixtyninestudios.kublit.yml

```diff
@@ -0,0 +1,38 @@
+app-id: org.sixtyninestudios.kublit
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: Kublit.sh
+finish-args: 
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
+  - --device=dri
+modules:
+  - name: Kublit
+    buildsystem: simple
+    build-commands:
+      - install -D Kublit.sh /app/bin/Kublit.sh
+      - install -D kubliticon.png /app/share/icons/kubliticon.png
```

Comment:
this is not where icons are installed. Nor even the proper name.

Reply:
https://github.com/flathub/com.sweethome3d.Sweethome3d has the same path for its icons

Reply:
two wrongs don't make a right.

Reply:
Where is it suppose to go then.

Reply:
https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html#directory_layout

e.g. `/app/share/icons/hicolor/32x32/apps/your.app.id.png`